projects/packages/paypal-payments/src/block/editor.scss

```diff
@@ -0,0 +1,146 @@
+@import '@automattic/jetpack-base-styles/gutenberg-base-styles';
```

Comment:
Since you're using this package, you'll need to add it as a dev dependency.

Reply:
I added it as a dev dependency in [515815e](https://github.com/Automattic/jetpack/pull/43413/commits/515815e86ee65b6388567ec3e481062a85966c84).